k8s/serviceaccount.go

```diff
@@ -0,0 +1,31 @@
+package k8s
+
+import (
+	"bytes"
+
+	"k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+func UpdateServiceAccount(k8sClient *kubernetes.Clientset, serviceAccountYaml string) error {
```

Comment:
can we include the fact that we are decoding yaml files in the function name something like `UpdateServiceAccountFromYaml`